ambry-clustermap/src/main/java/com.github.ambry.clustermap/FixedBackoffResourceStatePolicy.java

```diff
@@ -78,7 +78,7 @@ public void onHardDown() {
    */
   @Override
   public void onHardUp() {
-    logger.info("Marking resource {} as Hard up", resource);
+    logger.trace("Marking resource {} as Hard up", resource);
```

Comment:
This gets called once for every node when a live instance change trigger comes in, so it is better as trace.